### DISCUSSION TITLE
Welcome to nexus-finance-core Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/nexus-finance-core#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Clawue884** [comment]
Perkenalkan saya cakar884(clawue)

Ini merupakan proyek yang sangat di harapkan bagi komunitas,,di harapkan platform ini akan segera rilis,,karena akan sangat membantu komunitas....terimaksih banyak
